### DISCUSSION TITLE
Power Zone definition & color

### DISCUSSION THREAD

**Discussion by TClin76** [dvmarinoff/Auuki#86] [Ideas]
Hi,
Just found a minor issue that the color used in workout interval is a little bit weird. 
Color for zone-three is gray, which is the same as zone-one.

Another suggestion is that, the definition of zone range is also different from Coggan's power zones (or Zwift's zones)
It seems that the default zone percentages defined in Flux is 0.55/ 0.76/ 0.88/ 0.95/ 1.06/ 1.20.
And the zone percentages is defined a6/ 0.76/ 0.91/ 1.06/ 1.21/ 1.50 in Coggan's power zones. (Zwift is similar)
Just for your reference. THank you

**Comment by dvmarinoff** [comment]
This is mostly by design, but I agree it is not ideal, and can be potentially confusing.
See I prefer the TrainerRoad power zones [definitions](https://www.trainerroad.com/blog/cycling-power-zones-training-zones-explained/), which are listed in here:

<img width="761" alt="Screenshot 2021-12-22 at 9 25 30" src="https://user-images.githubusercontent.com/7523832/147062233-44854b56-4cce-49be-b562-01e3e4a2a357.png">

But still want to keep the Zwift colour scheme. Zwift has the Tempo and the Sweet Spot zone mixed together in green colour, but given the importance of the Sweet Spot area for training I prefer it is in distinct colour. I didn’t came up with a specific one for the Tempo zone that I like so I decided to move on to other more urgent tasks, and left it grey, which kinda also servers as a symbolic reminder not to spent too much time in this zone.

I am open for suggestion, if you can find I colour value (hex) that fits well with the others, or maybe a whole colour scheme for the zones, and I like it, I can add it.

**Comment by TClin76** [comment]
Understand! Thanks for the information. I haven't noticed that the zone definition in TrainerRoad is different.
![image](https://user-images.githubusercontent.com/96434118/147065204-c258a6a6-bec8-442c-af99-4b1ab6d4fcda.png)
For Zone3~Zone5, the color zone is not quite evenly distributed, so it is sort of weird to me. 
(it is about 15% in each power zone in Coggan's power zone definition)
However it is not a problem, keep the definition of TrainerRoad here works perfectly too.

Regarding to color value used for Tempo, how about the average value of Green (#57C057) and Blue (#328AFF)? which is #44A5AB.
![image](https://user-images.githubusercontent.com/96434118/147073835-d711ec58-517b-415f-970a-7e5cbb90ff20.png)

**Comment by dvmarinoff** [comment]
Added the teal colour to the development version it looks good to me:

<img width="749" alt="Screenshot 2021-12-23 at 14 44 43" src="https://user-images.githubusercontent.com/7523832/147242314-b5f668c5-fb5a-4614-86cb-6a7e907d4109.png">

It can be tested with this workout, which has the upper and lower bounds of the training zones as intervals:

```
<workout_file>
    <author>Flux</author>
    <name>Zone Colour Test</name>
    <category>Test</category>
    <description>Zone colour test.</description>
    <sportType>bike</sportType>
    <tags>
    </tags>
    <workout>
        <SteadyState Duration="60" Power="0.54" />

        <SteadyState Duration="60" Power="0.55" />
        <SteadyState Duration="60" Power="0.75" />

        <SteadyState Duration="60" Power="0.76" />
        <SteadyState Duration="60" Power="0.87" />

        <SteadyState Duration="60" Power="0.88" />
        <SteadyState Duration="60" Power="0.94" />

        <SteadyState Duration="60" Power="0.95" />
        <SteadyState Duration="60" Power="1.05" />

        <SteadyState Duration="60" Power="1.06" />
        <SteadyState Duration="60" Power="1.20" />

        <SteadyState Duration="60" Power="1.21" />
    </workout>
</workout_file>
```
